hypershift-operator/controllers/hostedcluster/hostedcluster_controller.go

```diff
@@ -1495,12 +1495,25 @@ func (r *HostedClusterReconciler) reconcile(ctx context.Context, req ctrl.Reques
 		return ctrl.Result{}, fmt.Errorf("failed to reconcile control plane operator: %w", err)
 	}
 
+	configOperatorImage, releaseVersion, err := func() (string, string, error) {
+		releaseInfo, err := r.ReleaseProvider.Lookup(ctx, hcluster.Spec.Release.Image, pullSecretBytes)
+		if err != nil {
+			return "", "", fmt.Errorf("failed to lookup release image: %w", err)
+		}
+		return releaseInfo.ComponentImages()["cluster-config-operator"], releaseInfo.Version(), nil
+	}()
+	if err != nil {
+		return ctrl.Result{}, fmt.Errorf("failed to get cli image: %w", err)
+	}
+
 	// Reconcile the Ignition server
 	if !controlplaneOperatorManagesIgnitionServer {
 		if err := ignitionserverreconciliation.ReconcileIgnitionServer(ctx,
 			r.Client,
 			createOrUpdate,
+			releaseVersion,
```

Comment:
In modern hypershift, this path is never taken (4.11 and later,) so we are doing the `Lookup` unnecessarily.  It is possible that IBM is still using 4.10 CPO.

Either way, we should move the `Lookup` inside this `if` block so we only do it if the HO is reconciling the ignition server, which is (almost) never.